doc/source/whatsnew/v0.17.0.txt

```diff
@@ -119,6 +119,7 @@ Other API Changes
 - Enable serialization of lists and dicts to strings in ExcelWriter (:issue:`8188`)
 - Allow passing `kwargs` to the interpolation methods (:issue:`10378`).
 - Serialize metadata properties of subclasses of pandas objects (:issue:`10553`).
+- Allow multi-indexes columns to be written to Excel with a one way serialization (:issue: `10564`).
```

Comment:
more like:

this was changed in 0.16.2 as the read-back method could not always guarantee perfect fidelity.